README.rst

```diff
@@ -0,0 +1,37 @@
+Pocketlint is an addon module for pylint.
```

Comment:
Pocketlint is doing much more. It's basically wrapping Pylint and changing how it is configured.